contents/blog/workflows-alpha.md

```diff
@@ -0,0 +1,106 @@
+---
+date: 2025-11-03
+title: Workflows Are Now in Alpha. And I Already Managed to Break Mine
+author:
+  - sara-miteva
+featuredImage: >-
+  
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: PostHog news
+---
+
+Monday morning. This is me, happy and caffeinated, walking to my coworking space, feeling productive and ready for the week.
+
+[Add image here, art request is submitted]
+
+Last week I had set up a new automation using our new **workflows** product (soon coming to beta).  
+The goal: test if everything works – and, at the same time, let users know it’s available to try.  
+I sent out a simple test email.  
+It looked like this:
+![workflwows email](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha1_04ea076a13.png)
+---
+
+I was genuinely excited to see how users would react.  
+Maybe some quick replies. Maybe some feedback.
+And… I got replies all right.  2
+But not quite the kind I expected.
+![reply to my email 1](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha3_8f3453dfbd.png)
+![reply to my email 2](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha2_dcc58991c0.png)
+![reply to my email 3](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha4_4174b14490.png)
```

Comment:
I'd probably replace with a single image that groups them. Makes it easier to read with a narrow layout. 

Quick version: https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/grouped_test_f1ecac3667.png